README.md

```diff
@@ -155,4 +162,4 @@ If you like this project please show support by staring ⭐️.
 
```

Comment:
- [ ] As mentioned by the previous reviewer in their summary kindly make sure the link under the Contributing  section in the readme file is leading to the issue page instead, kindly fix that 🙃

Reply:
Thank you @cynthiainga. I updated the link